src/main/java/com/minecraftabnormals/abnormals_core/common/blocks/wood/WoodPostBlock.java

```diff
@@ -0,0 +1,49 @@
+package com.minecraftabnormals.abnormals_core.common.blocks.wood;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.block.IWaterLoggable;
+import net.minecraft.fluid.FluidState;
+import net.minecraft.fluid.Fluids;
+import net.minecraft.item.BlockItemUseContext;
+import net.minecraft.state.BooleanProperty;
+import net.minecraft.state.StateContainer;
+import net.minecraft.state.properties.BlockStateProperties;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.shapes.ISelectionContext;
+import net.minecraft.util.math.shapes.VoxelShape;
+import net.minecraft.world.IBlockReader;
+
+public class WoodPostBlock extends Block implements IWaterLoggable {
+	private static final VoxelShape SHAPE = Block.makeCuboidShape(6F, 0F, 6F, 10F, 16F, 10F);
+	public static final BooleanProperty WATERLOGGED = BlockStateProperties.WATERLOGGED;
```

Comment:
Why is this public? It's not called externally anywhere.
```suggestion
	private static final BooleanProperty WATERLOGGED = BlockStateProperties.WATERLOGGED;
```